addons/medical_treatment/ACE_Medical_Treatment.hpp

```diff
@@ -566,7 +566,10 @@ class ADDON {
         // How long until the maximum effect is reached
         timeTillMaxEffect = 30;
         // How many of this type of medication can be in the system before the patient overdoses?
-        maxDose = 4;
+        maxSafeDose = 4;
```

Comment:
> `maxDose` -> `maxSafeDose`

No - I believe that config entry has been there for at least 4 years (since the medical rewrite came out, which is 4 years and nearly a 1 month now, though it might have been available for longer than that). We are going to keep things backwards compatible.